pageserver/src/tenant/remote_timeline_client/index.rs

```diff
@@ -236,6 +244,7 @@ mod tests {
                 0, 0,
             ]
             .to_vec(),
+            deleted_at: None,
```

Comment:
Can PR an example for the deleted_at.

Reply:
Don't get what you mean by this.
`deleted_at` is set to `Some()` by the `stop()` method.

Reply:
This was to have a test with `deleted_at: Some("...")`, an "image based" test, catalog of live versions we need to support (to be pruned when we can drop the support).

Reply:
Turned into follow-up: https://github.com/neondatabase/neon/issues/4162